### ISSUE TITLE
[BUG] make image-tornjak-manager leading to error

### ISSUE THREAD

**Issue by rooi6** [spiffe/tornjak#438]
The command, make image-tornjak-manager, leads to the following error on my Windows device:

Downloading go modules…
make: go: No such file or directory
make: *** [Makefile:45: download] Error 127

Steps:
I run wsl (Ubuntu) after installing wsl
cd into tornjak folder
run: sudo apt install make
run:  make image-tornjak-manager

Note: I did the same with make image-tornjak-backend and make image-tornjak-frontend and both ran successfully

**Comment by maia-iyer** [comment]
Good catch! I created #440 which would be an issue that resolves this problem. 

The core issue here is that golang needs to be installed locally. However, we shouldn't really let the build happen locally anyways and should be within a Docker container as is for the backend and frontend
